src/Html.php

```diff
@@ -1635,20 +1650,20 @@ public static function cssStyleToArray(string $style): array
      * - `[0]dates[0]` represents the first array element of the "dates" attribute for the first model in tabular
      *    input.
      *
-     * If `$attribute` has neither prefix nor suffix, it will be returned back without change.
      * @param string $attribute the attribute name or expression
-     *
-     * @return string the attribute name without prefix and suffix.
-     *
+     * @return array
      * @throws InvalidArgumentException if the attribute name contains non-word characters.
      */
-    public static function getAttributeName(string $attribute): string
+    public static function parseAttribute(string $attribute)
```

Comment:
This method looks like internal thing. Does it have to be `public`?

Reply:
i'm use this here: https://github.com/yiisoft/form/pull/48/files#diff-367552c756b09b53720eccec0ddf71a4R79

Reply:
Hm... maybe make method `Html::wrapAttributeName(string $wrapName, string $attributeName)` and make `parseAttribute` is private:

```php
`Html::wrapAttributeName('myform', 'name[0]'); // myform[name][0]
```

?

Reply:
I mean it's not used externally (except test) so likely it is a good idea to make it `private`.

Reply:
`parseAttribute` use in yiisoft/form in new PR: https://github.com/yiisoft/form/pull/48/files#diff-367552c756b09b53720eccec0ddf71a4R79

But now I make `parseAttribute` is private and create new method `wrapAttributeName`.